caffe2/CMakeLists.txt

```diff
@@ -463,7 +463,7 @@ if (NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     ${TORCH_SRC_DIR}/csrc/jit/vararg_functions.cpp
     )
 
-  if (NOT INTERN_BUILD_MOBILE)
```

Comment:
I think the previous line was pretty confusing - why disabling jit/mobile/* code for mobile build?!
I think the reason is that the code is still under development so we don't want to include it in actual mobile build to increase library size. While for non-mobile build we could include it to capture build error.

This PR adds mobile-interpreter sources to host build which simulates mobile build (which is used for code analysis purpose as well). It should not change the behavior for other types of build.